client/src/dummy.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
@prateekinstillion , try and avoid commiting `dummy` files to the main repo, and focus on the tests to make sure things are working as expected.

Reply:
OK. noted